src/tests/test_database.py

```
@@ -747,4 +782,270 @@ def test_link_task_label_db_connection_error(self):
+    def tearDown(self):
+        self._cleanup_log_dirs()
+        super().tearDown()
+
+    def _cleanup_log_dirs(self):
+        """Helper method to clean up log directories."""
+        for log_dir in [self.default_log_dir, self.user_log_dir]:
+            if os.path.exists(log_dir):
+                try:
+                    os.rmdir(log_dir)
+                except OSError:
+                    pass  # Directory might not be empty or might be used by other processes
```

Ruff (0.8.2)

Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

(SIM105)